addon/components/base-layer.js

```diff
@@ -98,6 +100,27 @@ export default Ember.Component.extend(
     }),
 
     /**
+      This layer's opacity
+      @property opacity
+      @type Float
```

Comment:
As I know JS doesn't have such types as 'Float', 'Int', 'Double' and so on, it has only 'Number' type.